File: QuickStart/RibbonBar/ResizingDemo/MyFlowLayoutPanel.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Windows.Forms;

namespace Telerik.Examples.WinControls.RibbonBar.ResizingDemo
{
    class MyFlowLayoutPanel : FlowLayoutPanel
    {
        public MyFlowLayoutPanel()
        { 
            this.SetStyle(ControlStyles.DoubleBuffer, true);
            this.SetStyle(ControlStyles.AllPaintingInWmPaint, true);
            this.SetStyle(ControlStyles.UserPaint, true);
            this.SetStyle(ControlStyles.SupportsTransparentBackColor | ControlStyles.OptimizedDoubleBuffer, true);
        }
    }
}

File: QuickStart/RibbonBar/ResizingDemo/MyFlowLayoutPanel.vb
Imports System.Windows.Forms

Class MyFlowLayoutPanel
    Inherits FlowLayoutPanel
    Public Sub New()
        Me.SetStyle(ControlStyles.DoubleBuffer, True)
        Me.SetStyle(ControlStyles.AllPaintingInWmPaint, True)
        Me.SetStyle(ControlStyles.UserPaint, True)
        Me.SetStyle(ControlStyles.SupportsTransparentBackColor Or ControlStyles.OptimizedDoubleBuffer, True)
    End Sub
End Class